syntax = "proto3";

option java_multiple_files = true;
option java_package = "com.lxl.demo.grpc";
option java_outer_classname = "DemoProto";
option objc_class_prefix = "LXL";

package demo;

service Greeter {
  rpc SignUP (DemoRequest) returns (DemoReply) {}
  rpc SignIn (DemoRequest) returns (DemoReply) {}
}

message DemoRequest {
  string username = 1;
  string pwd = 2;
}

message DemoReply {
  string message = 1;
}
